tools/pytest.ini

```diff
@@ -1,9 +1,28 @@
 [pytest]
-# Directories with their own tox.ini: wave, wpt, wptrunner
-# Python 3 only: quic (it should have its own tox.ini eventually)
-norecursedirs = .* {arch} *.egg third_party wave wpt wptrunner quic
+norecursedirs = .* {arch} *.egg third_party
```

Comment:
Does this not change which tests are run?

Reply:
No; see also the `conftest.py` change which excludes everything with its own `tox.ini` from being recursed into.